wafer/users/views.py

```diff
@@ -53,3 +56,11 @@ class EditProfileView(EditOneselfMixin, UpdateView):
 
     def get_success_url(self):
         return reverse('wafer_user_profile', args=(self.object.user.username,))
+
+
+class UserViewSet(viewsets.ModelViewSet):
+    """API endpoint for users."""
+    queryset = get_user_model().objects.all()
+    serializer_class = UserSerializer
+    # We want some better permissions than the default here, but
+    # IsAdminUser will do for now.
```

Comment:
We don't seem to actually set the permissions to `IsAdminUser` here? The docs seem to indicate that the permissions might default to `AllowAny` unless configured otherwise in settings? Either way, seems good to be explicit.